ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.scss

```diff
@@ -458,52 +458,67 @@
           text-overflow: ellipsis;
       }
 
-      .dataTable tbody td div.triple-cell ul.triple-list {
-        margin-bottom: 0;
-      }
-
       /** Shows copy resource link when mouse is over a table row with resources of type uri or triple */
-      .dataTable tbody td div.uri-cell:hover .resource-copy-link,
-      .dataTable tbody td div.triple-cell ul.triple-list li:hover .resource-copy-link,
-      .dataTable td div.triple-cell div.triple-close:hover .resource-copy-link {
+      .dataTable tbody td div.uri-cell:hover .resource-copy-link {
         display: inline;
       }
 
-      .dataTable tbody td div.uri-cell:hover,
-      .dataTable tbody td div.triple-cell ul.triple-list li:hover,
-      .dataTable td div.triple-cell div.triple-close:hover {
+      .dataTable tbody td div.uri-cell:hover {
         .spacer {
           display: none;
         }
       }
 
-      .dataTable td div {
-        .spacer {
-          min-width: 1em;
+      .dataTable tbody tr td .triple-cell {
+        display: flex;
+        flex-direction: column;
+        align-items: stretch;
+
+        .triple-link {
+          color: #770088;
+          text-decoration: none;
+        }
+
+        .triple-list {
+          display: flex;
+          flex-direction: column;
+
+          .uri-link {
+            padding-left: 20px;
+          }
         }
       }
 
-      .dataTable tbody td div.triple-cell ul.triple-list li {
-        list-style-type: none;
+      .dataTable tbody tr td div.triple-cell div.triple-close-link,
```

Comment:
Part of the selectors in this section of the stylesheet are nested, like in the above fragment, and others don't. Some css classes are augmented with the additional tag name, others don't. I appreciate the cleanup you did, it was necesarry, but I think it lacks a bit of consistency.